packages/ember-metal/lib/index.js

```diff
@@ -275,30 +275,6 @@ Ember.isGlobalPath = isGlobalPath;
 
 Ember.run = run;
 
-/**
-@class Backburner
-@for Ember
-@private
-*/
-Ember.Backburner = function() {
-  deprecate(
-    'Usage of Ember.Backburner is deprecated.',
-    false,
-    {
-      id: 'ember-metal.ember-backburner',
-      until: '2.8.0',
```

Comment:
This one needs to stay.  `until` is inclusive (meaning we have to keep it around through 2.8.0).  Due to the LTS policy this needs to stick around until we release another LTS version (which will be 2.8).